src/qml/LayerLoginDialog.qml

```diff
@@ -58,7 +58,7 @@ Page {
       id: username
       Layout.alignment: Qt.AlignHCenter | Qt.AlignVCenter
       Layout.preferredWidth: Math.max( parent.width / 2, usernamelabel.width )
-      inputMethodHints: Qt.ImhPreferLowercase
+      inputMethodHints: Qt.ImhNoPredictiveText | Qt.ImhNoAutoUppercase | Qt.ImhPreferLowercase
```

Comment:
Is nopredictivetext correct here? I often autofill with emailaddress.

Reply:
Yeah it just prevents auto completion, you'll still see it within your virtual keyboard.